README.md

```diff
@@ -1 +1,292 @@
+  return 'Hola'
+}
+
+exports.sayHelloInJapanese = function() {
+  return 'こんにちは' // Kon'nichiwa
+}
+```
+
+That's great! In this case `exports` and `module.exports` are the same. What if we want to assign the whole object like we did with the configuration module to the `exports` object?
+
+So this is the working code with `module.exports`. Obviously the code with `module.exports` works, because we used it in the previous section:
+
+```js
+module.exports = function(env) {
+  env = env || process.env.NODE_ENV
+  if (env == 'production') {
+    ...
+  } else if (env == 'testing') {
+    ...
+  } else { // Assume development
+    ...
+  }
+}
+```
+
+What about this code with `exports`? Will it work too?
+
+```js
+exports = function(env) {
+  env = env || process.env.NODE_ENV
+  if (env == 'production') {
+    ...
+  } else if (env == 'testing') {
+    ...
+  } else { // Assume development
+    ...
+  }
+}
+```
+
+Go ahead and try to run it (`environment-exports.js`) with this one-liner command:
+
+```
+node -e "console.log(require('./environment-exports')().apiKey)"
+```
+
+If you're getting `TypeError: require(...) is not a function`, that's right. You are getting an error because now nothing has been exported. By writing `exports = ...`, we **lost** the original value of `exports`!
+
+Note: We lose the reference, because we overwrite the object `exports`. It's okay to overwrite `module.exports` because in this case `module.exports` is a property of the `module` object.
+
+Let's take another example of code to refactor. The code below works, but what happens if we just try to use `exports` instead of `module.exports`?
+
+With `module.exports`:
+
+```js
+module.exports = {
+  sayHelloInEnglish: function() {
+    return 'Hello'
+  },
+  sayHelloInTatar: function() {
+    return 'Isänmesez'
+  },
+  sayHelloInSpanish: function() {
+    return 'Hola'
+  },
+  sayHelloInJapanese: function() {
+    return 'こんにちは' // Kon'nichiwa
+  }
+}
+```
+
+With just `exports`:
+```js
+exports = {
+  sayHelloInEnglish: function() {
+    return 'Hello'
+  },
+  sayHelloInTatar: function() {
+    return 'Isänmesez'
+  },
+  sayHelloInSpanish: function() {
+    return 'Hola'
+  },
+  sayHelloInJapanese: function() {
+    return 'こんにちは' // Kon'nichiwa
+  }
+}
+```
+
+So what's happening with just `exports =...`? It won't work as expected, because we cannot assign exports to something. We lose the reference.
+
+We need to use `module.exports =...` if we want to assign the entire module, i.e., if we want to have the logic as the result of `require()` and not as the result of `require().NAME` where `NAME` is some arbitrary name (e.g, `configs`, `users`).
+
+Typicall, Node developers have one `module.exports = ...` in a file or many `exports.NAME =...`. When to use `module.exports = ...` vs. `exports.name = ...` or `module.exports.name = ...`? In other words, when do you want to use `var name = require('name')` and when `var nameB = require('nameA').nameB`?
```

Comment:
I have a hard time understanding this paragraph. Could we skip the first sentence?